tests/drivers/build_all/sensor/prj.conf

```diff
@@ -98,4 +98,4 @@ CONFIG_TMP112=y
 CONFIG_TMP116=y
 CONFIG_VCNL4040=y
 CONFIG_VL53L0X=y
-CONFIG_INA23X=y
+CONFIG_INA230=y
```

Comment:
will this build without selecting `INA23X`?

Reply:
Right, I missed that. Fixed.